Updating the Databox package to the latest stable rest api spec version

GovardhanaDK opened the pull request:
<!-- DO NOT DELETE THIS TEMPLATE -->

## Description

<!-- Please add a brief description of the changes made in this PR. If you have an ongoing or finished cmdlet design, please paste the link below. -->
Moving to the latest stable version. This supports -SkuModel which is a necessary property for working with V2 SKUs
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [X] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

dolauli commented:
/azp run

GovardhanaDK commented:
/azp run

GovardhanaDK commented:
@dolauli I have made changes to the test case. Please do trigger the test pipelines now

dolauli commented:
/azp run

dolauli commented:
Please update the https://github.com/GovardhanaDK/azure-powershell/blob/main/src/DataBox/DataBox/ChangeLog.md accordingly otherwise your change will not be included in our next release.

GovardhanaDK commented:
@dolauli I have updated the version and ChangeLog.md file


The cred scan failure is because of 
![image](https://github.com/user-attachments/assets/4ce917e1-89f7-4e07-9fe1-b3d278004c5b)
This is a random password we are using for our tests to test the flow where the customer enters their own device password.

dolauli commented:
/azp run

dolauli commented:
> @dolauli I have updated the version and ChangeLog.md file
> 
> The cred scan failure is because of ![image](https://private-user-images.githubusercontent.com/36787828/435038530-4ce917e1-89f7-4e07-9fe1-b3d278004c5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ5NDc1MDEsIm5iZiI6MTc0NDk0NzIwMSwicGF0aCI6Ii8zNjc4NzgyOC80MzUwMzg1MzAtNGNlOTE3ZTEtODlmNy00ZTA3LTlmZTEtYjNkMjc4MDA0YzViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE4VDAzMzMyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM0NzBiMzlhM2VkNjljZDliNjZkYjA1NDZmMGQyNjU4MzFmOTc0MWU5MjUyYTdlZTI0ODFlZjYxMDhlMWY1MTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wDdzvmcSIZEULv1Te6wj4gNywlalr8iXcEoBBgNVUDg) This is a random password we are using for our tests to test the flow where the customer enters their own device password.

Change it to something like "******" when you create the PR

GovardhanaDK commented:
> > @dolauli I have updated the version and ChangeLog.md file
> > The cred scan failure is because of ![image](https://private-user-images.githubusercontent.com/36787828/435038530-4ce917e1-89f7-4e07-9fe1-b3d278004c5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ5NDc1MDEsIm5iZiI6MTc0NDk0NzIwMSwicGF0aCI6Ii8zNjc4NzgyOC80MzUwMzg1MzAtNGNlOTE3ZTEtODlmNy00ZTA3LTlmZTEtYjNkMjc4MDA0YzViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE4VDAzMzMyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM0NzBiMzlhM2VkNjljZDliNjZkYjA1NDZmMGQyNjU4MzFmOTc0MWU5MjUyYTdlZTI0ODFlZjYxMDhlMWY1MTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wDdzvmcSIZEULv1Te6wj4gNywlalr8iXcEoBBgNVUDg) This is a random password we are using for our tests to test the flow where the customer enters their own device password.
> 
> Change it to something like "******" when you create the PR

Changing it ****** will cause the test case to fail. I have added the two files under CredScanSupression as they are just test related file and the passwored is not an actual password.
The password is used to lock files on a device. Since there is no device involved here, the password is never actuallly used

dolauli commented:
/azp run
